en/exercises/device_behaviour_check/device_assessment_checklist.md

```diff
@@ -0,0 +1,125 @@
+
+* OS Security Updates
+  * Start --Settings --Update & Security --Windows Update
+* Firewall
+  * Start, type Firewall (select Windows Firewall)
+* Privacy
+  * Start --Settings -- Privacy 
+* Anti-Virus Version
+  * 
+* Privacy
+  * (GUI) Start --Settings -- Privacy
+* User privilege
+  * Start, type 'User Account', select "Change User Account Control settings" 
+* Drive Encryption
+  * (Bitlocker), https://docs.microsoft.com/en-us/windows/device-security/bitlocker/bitlocker-device-encryption-overview-windows-10
+* Services Running
+  * Start, type "Task Manager"
+
+
+**Windows 8**
+
+ * OS Security Updates
+ * Firewall
+   *  (GUI) Start (or Down Arrow Icon, PC Settings) -- Control Panel -- Windows Firewall
+   *  (CLI)
+```Netsh Advfirewall show allprofiles```
+   *  (more detail: http://windows.microsoft.com/en-us/windows-8/windows-firewall-from-start-to-finish)
+ * Anti-Virus Version
+   * 
+ * User privilege
+   * 
+ * Drive Encryption
+   * 
+   * https://diskcryptor.net/wiki/Main_Page
+ * Services Running
+   * Right-Click on bottom taskbar, select "Task Manager"
+
+Installed updates
+
+Control Panel Programs and features installed updates
+CLI:
+http://www.techsupportalert.com/en/quick-and-easy-way-list-all-windows-updates-installed-your-system.htm
+
+**Windows 7**
+
+In Windows 7, (GUI) Control Panel -- All Control Panel Items -- Action Center (Security tab) provides a quick run-down of most security features installed and their update status. It does not show drive encryption or specific versions.
+
+ * OS Security Updates
+   * 
+ * Firewall
+   * (GUI) Control Panel -- All Control Panel Items -- Windows Firewall
+   * (CLI) 
+```Netsh Advfirewall show allprofiles```
+ * Anti-Virus Version
+   * 
+ * User privilege
+   * (GUI) Control Panel -- All Control Panel Items -- User Accounts and checking also the User Account Control settings.
+ * Drive Encryption
+   * (GUI) Control Panel -- All Control Panel Items -- BitLocker Drive Encryption
+   * (TrueCrypt) , https://diskcryptor.net/wiki/Main_Page 
+ * Services Running
+   * (CLI)
+```tasklist```
+   * (GUI) Right-click on task bar, select "Start Task Manager"
+   * (Advanced) Use TechNet/SysInternal's Process Explorer: http://technet.microsoft.com/en-us/sysinternals/bb896653.aspx 
+
+**Windows XP**
+
+If user is still operating on windows XP, recommendation is to upgrade to later windows. Why you should not use Windows XP https://www.microsoft.com/windows/en-us/xp/end-of-xp-support.aspx
+
+
+##### Linux
+ * Firewall
+   * (CLI) 
+```sudo iptables -L -n```
+   * (CLI)   (Ubuntu, and only if installed)
+```sudo ufw status```
+   * (GUI) (Ubuntu, and only if installed)
+```gufw```
+ * Anti-Virus Version
+   * (CLI) deb: 
+```dpkg-query -l | grep virus``` rpm: ```yum list installed | grep virus```
+   * See also: https://en.wikipedia.org/wiki/Linux_malware#Anti-virus_applications
+ * User privilege
+   * (CLI) 
+```groups```
+
+ * Drive Encryption
+   * 
+   * (TrueCrypt)
```

Comment:
I know this is an assessment, but there is no TrueCrypt anymore, similar to Windows XP let's change this to (VeraCrypt) or the default distro-specific full desk encryption like LUKS. See the following: https://help.ubuntu.com/community/ManualFullSystemEncryption